packages/ipfs/test/core/preload.spec.js

```diff
@@ -2,9 +2,10 @@
 'use strict'
 
 const { nanoid } = require('nanoid')
+const { Buffer } = require('buffer')
 const { expect } = require('interface-ipfs-core/src/utils/mocha')
 const all = require('it-all')
-const MockPreloadNode = require('../utils/mock-preload-node')
+const MockPreloadNode = require('../utils/mock-preload-node-utils')
```

Comment:
Should this not be `const MockPreloadNode = require('../utils/mock-preload-node')` ?

Reply:
i split the server from the helpers so we don't include the server stuff in the tests bundle